src/utils/cpUtils.ts

```diff
@@ -34,8 +39,18 @@ export namespace cpUtils {
             childProc.on('error', reject);
             childProc.on('close', (code: number) => {
                 if (code !== 0) {
-                    reject(new Error(localize('azFunc.commandError', 'Command "{0} {1}" failed with exit code "{2}".', command, args.toString(), code)));
+                    if (outputChannel) {
+                        outputChannel.show();
+                        reject(new Error(localize('azFunc.commandErrorWithOutput', 'Failed to run "{0}" command. Check output window for more details.', command)));
```

Comment:
Not sure I'm understanding why having an output channel or not should affect the amount of detail you display.

Reply:
In both cases we're showing the same amount of details. See the 'after' screenshots for both scenarios. They have similar amounts of details. The only difference is where we display those details - from this file or in AzureActionHandler